static/script/devices/mediaplayer/seekfinishedemitevent.js

```diff
@@ -0,0 +1,56 @@
+require.def(
```

Comment:
This needs a license header comment at the top.

It could also really really use a comment explaining what its for and why!